perllib/FixMyStreet/App/Controller/Around.pm

```diff
@@ -165,10 +165,27 @@ sub display_location : Private {
     $c->stash->{all_pins} = $all_pins;
     my $interval = $all_pins ? undef : $c->cobrand->on_map_default_max_pin_age;
 
+    my $states;
+    $c->stash->{filter_status} = $c->cobrand->on_map_default_status;
```

Comment:
I think this default setting would be clearer as an `else` in the below `if` section.